src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1294,6 +1294,7 @@ namespace ReportSizingManager {
                         CoilInHumRat = FinalZoneSizing(CurZoneEqNum).DesCoolCoilInHumRat;
                         DesCoilLoad = FinalZoneSizing(CurZoneEqNum).DesCoolMassFlow *
                                       (PsyHFnTdbW(CoilInTemp, CoilInHumRat) - PsyHFnTdbW(CoilOutTemp, CoilOutHumRat));
+                        CoilDesWaterDeltaT = DataWaterCoilSizCoolDeltaT; // Temporary fix by Gu on 7/19/18
```

Comment:
Temporary fix by Gu?

Reply:
@Myoldmopar @lgu1234 This line can be removed now, then merge in develop.  It was fixed in #6850.

Reply:
@mjwitte @Myoldmopar I am going to remove it.